src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -59,32 +59,62 @@ class Recent_Past_View extends View {
 		'components/events-bar/search',
 		'components/events-bar/filters',
 		'components/events-bar/views',
+		'components/ical-link',
+		'components/breakpoints',
 
 		// Day View
 		'day',
+		'day/top-bar',
+		'day/top-bar/nav',
+		'day/top-bar/nav/prev',
+		'day/top-bar/nav/next-disabled',
+		'day/top-bar/datepicker',
 
 		// List View.
 		'components/events-bar/views/list',
 		'components/events-bar/views/list/item',
 		'list/top-bar',
 		'list/top-bar/nav',
+		'list/top-bar/nav/prev',
+		'list/top-bar/nav/next-disabled',
 		'list/top-bar/datepicker',
 		'list',
 
 		// Month View
 		'month',
+		'month/top-bar',
+		'month/top-bar/nav',
+		'month/top-bar/nav/prev',
+		'month/top-bar/nav/next-disabled',
+		'month/top-bar/datepicker',
 
 		// Map View
 		'map',
+		'map/top-bar',
+		'map/top-bar/nav',
+		'map/top-bar/nav/prev',
+		'map/top-bar/nav/next-disabled',
+		'map/top-bar/datepicker',
 
 		// Photo View
 		'photo',
+		'photo/top-bar',
+		'photo/top-bar/nav',
+		'photo/top-bar/nav/prev',
+		'photo/top-bar/nav/next-disabled',
+		'photo/top-bar/datepicker',
 
 		// Week View
 		'week',
+		'week/top-bar',
+		'week/top-bar/nav',
+		'week/top-bar/nav/prev',
+		'week/top-bar/nav/next-disabled',
+		'week/top-bar/datepicker',
```

Comment:
should these be filtered from pro?

Reply:
Since here we're just filtering, and not having PRO active will not have an effect, I think these could live here.